### ISSUE TITLE
Regarding validation transforms

### ISSUE THREAD

**Issue by MukundVarmaT** [AllenXiangX/SnowflakeNet#13]
Hi,
I noticed that you are scaling the point clouds by 0.85 during validation - [#](https://github.com/AllenXiangX/SnowflakeNet/blob/main/utils/data_loaders.py#L244). Wouldn't this reduce the chamfer value as the whole object is now compressed to a smaller size? To confirm this I computed the chamfer distance with and without it on the validation set using the provided pre-trained model (c3d).

Validation transforms as is - CD: 0.000872 (is as expected and close to the reported numbers on the test set)
Using the test set transforms during validation (i.e only ToTensor()) - CD: 0.001968 (more than twice the previous case)

It would be really helpful if you could clarify the same.
Thanks

**Comment by AllenXiangX** [comment]
Hi，
The scaling is an augmentation step that helps the network to converge faster on the c3d dataset. And only the training and validation set of c3d is scaled, which will not affect the result on the test set of c3d or the PCN dataset.

**Comment by MukundVarmaT** [comment]
@AllenXiangX 
Thank you for your prompt response. This is actually quite surprising in the sense a model that would obtain around 19x10^-4 would achieve close to 7x10^-4 on the test set? Is there such a huge difference in these two splits? 

I would really appreciate if you could let me know your thoughts on this. Thanks

**Comment by AllenXiangX** [comment]
I'm not sure about the difference. You can train the network without scaling and check its performance on these two splits, or it's better to use the PCN dataset for experiments, which I believe is larger and more stable.
